app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2972,4 +2991,64 @@ private function getExistingSku($sku)
     {
         return $this->_oldSku[strtolower($sku)];
     }
+
+    /**
+     * Set product images 'disable' = 0 for specified store.
+     *
+     * @param array $mediaGalleryData
+     * @return array
+     */
+    private function restoreDisableImage(array $mediaGalleryData)
```

Comment:
I know we already have a massive file here but would it make sense/be possible to extract the image processing to a separate class?

Reply:
Done.